liquibase-service/src/main/resources/db/rdb_modern/routines/013-sp_hepatitis_datamart_postprocessing-003.sql

```diff
@@ -0,0 +1,2609 @@
+CREATE OR ALTER PROCEDURE dbo.sp_hepatitis_datamart_postprocessing
+    @phc_id nvarchar(max),
+    @pat_ids nvarchar(max) = '',
```

Comment:
@upasanapattnaik-eq why @pat_ids are defaulted to empty string? Do we have a logic to look for empty string before it is used in downstream sql?

Reply:
It is not used downstream as of right now.  The service sends both phc_id and associated pat ids. 
As an improvement ticket,  both will be incorporated to enforce updates/inserts to Hepatitis datamart.